### ISSUE TITLE
Release to npm

### ISSUE THREAD

**Issue by appwiz** [awslabs/aws-sdk-react-native#2]
Release the AWS SDK components thru npm.

**Comment by jfo84** [comment]
So in the mean time I download the package and npm pack/install from the folders?

**Comment by appwiz** [comment]
@jford8820 yes, that's the current approach. I'm actively working on getting these packages into npm.

**Comment by jfo84** [comment]
Great. Thanks @appwiz. Love the work yall are doing but I'm sticking to a general uploader for now

**Comment by lielran** [comment]
why not release it as npm package? do you need a help with that?

**Comment by philipheinser** [comment]
@appwiz Is there a timeline when the package will be available on npm?

**Comment by tafelito** [comment]
Hi, I did npm pack inside Core and then npm install in teh services but I get `aws-sdk-react-native-transfer-utility@0.0.1 requires a peer of aws-sdk-react-native-core@>= 0.0.1 but none was installed` 

I couldn't make it work at all with RN version >=0.37. 

If you can improve the docs it would be great! 

Thank you

**Comment by mjmaix** [comment]
try `npm install ../Core/aws-sdk-react-native-core-0.0.1`

**Comment by sibelius** [comment]
@appwiz any progress on this, this would make easy to everybody to test and user this package
